Classes/DDFileLogger.m

```diff
@@ -136,6 +136,10 @@ - (void)dealloc {
     @try {
         [self removeObserver:self forKeyPath:NSStringFromSelector(@selector(maximumNumberOfLogFiles))];
         [self removeObserver:self forKeyPath:NSStringFromSelector(@selector(logFilesDiskQuota))];
+        __auto_type path = [NSTemporaryDirectory() pathComponents];
```

Comment:
These four values are unused.

Reply:
@ffried 
Oh! It was a mistake. I just use it to show "the blessing of __auto_type" in different PR.